compose.yml

```diff
@@ -197,21 +198,21 @@ services:
         entrypoint:
             - /updateEFO/updateEFO.sh
 
-    restic:
+    restic_s3:
         image: ghcr.io/wikiteq/restic-backup-docker:stream
-        env_file:
-            - ./.env
-            - ./.env.secret.restic
         profiles:
             - production
         restart: unless-stopped
         links:
             - db
         environment:
             - MW_DB_USER=root
-            - MW_DB_PASS=$MW_DB_INSTALLDB_PASS
+            - MW_DB_PASS=/run/secrets/db_root_password
             - MW_DB_NAME=$MW_DB_NAME
+            - RESTIC_REPOSITORY_FILE=/run/secrets/restic_s3_repository
+            - RESTIC_PASSWORD_FILE=/run/secrets/restic_password
             - RESTIC_TAG=backup_full
+            - AWS_SHARED_CREDENTIALS_FILE=/run/secrets/aws_credentials
```

Comment:
We should use `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` instead https://github.com/WikiTeq/restic-backup-docker?tab=readme-ov-file#environment-variables